tasks/interactive-bots.md

```diff
@@ -17,38 +17,38 @@
 ## Background
 
 Zulip [contrib-bots](https://github.com/zulip/zulip/tree/master/contrib_bots/lib)
-is a boilerplate for creating interactive bots that react to messages sent
-by users.
+ -is a boilerplate for creating interactive bots that react to messages
+ sent by users.
 
-The interactive bots live in the `contrib-bots/lib` as python files ('.py') that define
-their specific behavior. The `contrib-bots/run.py` file defines common behaviors
-for the interactive bots that react to messages.
+The interactive bots are python files in the contrib-bots/lib that define a 
+bot's specific behavior. The `contrib-bots/run.py` file defines common 
+behaviors for the interactive bots that react to messages.
 
-The following tasks will introduce you to using interactive bots and creating
-simple new bots that react to messages. This group of tasks has a high creative
-potential, as you can create your own bots that react to specific messages and
-can be integrated with various APIs.
+The following tasks will introduce you to using interactive bots and 
+creating new simple bots that react to messages. The skills taught in this 
```

Comment:
I'm not sure how rephrasing here improves the readability of the doc? The phrasing about creativity was intentional.